### PR TITLE
Pull request for comm_msg_validation

### PR DESCRIPTION
**Pull request by djantea**
### Contributor checklist
- [x] My commit messages follow the [Conventional Commit Message Format](https://gist.github.com/stephenparish/9941e89d80e2bc58a153#format-of-the-commit-message)
      For example: `fix: Multiply by appropriate coefficient`, or
      `feat(Calculator): Correctly preserve history`
      Any explanation or long form information in your commit message should be
      in a separate paragraph, separated by a blank line from the primary message
- [x] Bug fixes include regression tests
- [x] Features include unit/acceptance tests

### CODE REVIEW

File: lib/mix/tasks/git_ops.check_message.ex [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  @doc false
+  def run([path]) do
+    message = File.read!(path)
+
+    case Commit.parse(message) do
+      {:ok, _} ->
+        :ok
+
+      :error ->
```
Comment (zachdaniel) [inline lib/mix/tasks/git_ops.check_message.ex:27]: I'd really love to get more informative errors here, but for now this is the only way. Great.

**Comment by zachdaniel** [comment]
Alright, so far so good! The code looks great, haven't had a chance to run it. I'll set it up for one of my projects this weekend and give it a go, and then we can merge it!

**Comment by asummers** [comment]
Just a bump on this guy -- curious if this is good to go. /cc @zachdaniel

**Comment by zachdaniel** [comment]
Ack, gotta get on this. I've been planning on just trying it out in one of my projects to confirm, just haven't gotten around to it. I'll do it today!

**Comment by zachdaniel** [comment]
Sorry for the crazy delay!

**Review by zachdaniel — APPROVED** [review]

**Comment by djantea** [comment]
That is fine. I am pretty busy myself. Thanks.

**Comment by zachdaniel** [comment]
Released as 1.1.0

**Pull request merged**